pkg/csconfig/api.go

```diff
@@ -211,8 +214,25 @@ func (c *Config) LoadAPIServer() error {
 		log.Warning("crowdsec local API is disabled from flag")
 	}
 
-	if c.API.Server == nil {
-		log.Warning("crowdsec local API is disabled because its configuration is not present")
+	if c.API.Server != nil {
+		logLevel := log.InfoLevel
```

Comment:
shouldn't this be inherited from `api->server->log_level` or at least `common->log_level` ?